Source/Bogus/DataSets/Commerce.cs

```diff
@@ -14,11 +14,14 @@ public string Department(int max = 3, bool returnMax = false)
             var num = max;
             if( !returnMax )
             {
-                 num = this.Random.Number(max);
+                do
+                {
+                    num = this.Random.Number(max);
+                } while (num == 0);
```

Comment:
Hey, there! Thanks very much for your PR and bringing this up to my attention. I really appreciate it. You're absolutely correct, I can reproduce the the exception with `Generate(1000)`. I'm not sure a `do...while` loop is best here though. I think it can be simplified to: `num = this.Random.Number(1, max);`. The `Random.Number(` overload supports a minimum parameter.  The random value we received was zero, which ultimately selected zero categories.